source/Government.cpp

```diff
@@ -103,6 +103,8 @@ void Government::Load(const DataNode &node)
 			displayName = name;
 	}
 
+	bool notClearedRaids = true;
```

Comment:
No need for a double negative. Looking elsewhere, the variable would look like this instead:
```suggestion
	bool clearRaids = !raidFleets.isEmpty();
```